patches/chrome-browser-ui-views-bubble-webui_bubble_manager.h.patch

```diff
@@ -1,12 +1,21 @@
 diff --git a/chrome/browser/ui/views/bubble/webui_bubble_manager.h b/chrome/browser/ui/views/bubble/webui_bubble_manager.h
-index 69ea403a3a306b4d8bb50265c5a1d8ca4db05e26..c5afb3857893a356ccd15a5a0b236d96c1d44ea1 100644
+index ea2b2c5f0e7127677cbeb1a500ca778b544c7709..5815b27a3967fdd531f14fa024e9fe701bad2307 100644
 --- a/chrome/browser/ui/views/bubble/webui_bubble_manager.h
 +++ b/chrome/browser/ui/views/bubble/webui_bubble_manager.h
-@@ -202,6 +202,7 @@ class WebUIBubbleManagerT : public WebUIBubbleManager {
+@@ -146,7 +146,7 @@ class WebUIBubbleManagerImpl : public WebUIBubbleManager {
+         task_manager_string_id_(task_manager_string_id) {}
+   ~WebUIBubbleManagerImpl() override = default;
  
-     auto bubble_view = std::make_unique<WebUIBubbleDialogView>(
-         anchor_view_, contents_wrapper->GetWeakPtr(), anchor, arrow);
+- private:
```

Comment:
we should not be changing all private methods to protected. What specifically do we here?

Reply:
actually why aren't we just making these changes in `WebUIBubbleDialogView`? We can do a chromium_src override for `SetLayoutManager` and do the customization there. The current approach is extremely difficult to follow and seems unnecessarily complicated

Reply:
looking at this again I'm confused because we apparently have two different styles of webui bubbles? Why would we do that? Don't we want consistent styling of either always rounded corners or never? Why do only a few different things use BraveWebUIBubbleManager?

Reply:
cc @zenparsing

Reply:
also it's still not clear to me what is private here that needs to be protected

Reply:
To clarify about the `protected` patch, this patch was only introduced because we have a few places where we call `CreateWebUIBubbleDialog` inside the derived class, which is something upstream doesn't seem to want anymore. I was going to have @simonhong reviewing this, but at some point this fell through the things that needed to be checked.

Example error.

```
../../brave/browser/ui/views/brave_shields/cookie_list_opt_in_bubble_host.cc:103:52: error: 'CreateWebUIBubbleDialog' is a private member of 'WebUIBubbleManagerImpl<CookieListOptInUI>'
  103 |         WebUIBubbleManagerImpl<CookieListOptInUI>::CreateWebUIBubbleDialog(
      |                                                    ^
../../../src/chrome/browser/ui/views/bubble/webui_bubble_manager.h:152:40: note: declared private here
  152 |   base::WeakPtr<WebUIBubbleDialogView> CreateWebUIBubbleDialog(
      |              
```

Reply:
in either case we should definitely do a follow-up to look at this in general because it seems very odd to me that we have two different styles of bubbles

Reply:
Created a follow up issue https://github.com/brave/brave-browser/issues/37366

Reply:
Pushed 4769dccc28a6954c56795efc6327e071fd922531 which sets `CreateWebUIBubbleDialog` as the only protected method.

Reply:
It looks like `WebUIBubbleManager` has been significantly changed recently, which is not surprising since the API was not ideal. Hopefully our customizations can be simplified now.